### PR TITLE
Fixes for report.html

### LINKED ISSUE #34: Back button doesn't work in report

### PR DESCRIPTION
**Pull request by La0**
1. Display children nb
2. Use saner/simpler html update to avoid crashing on "back"

Fixes #34

### CODE REVIEW

File: report.js [hunk 1]
```diff
@@ -44,7 +44,11 @@ async function showDirectory(dir, files) {
     });
     output.appendChild(entryElem);
   }
-  document.getElementById('output').replaceWith(output);
```
Comment (marco-c) [inline report.js]: I wonder why replaceWith doesn't work here

Comment (marco-c) [reply report.js]: Maybe you can do:
```JavaScript
prev_output = document.getElementById('output')
prev_output.parent.replaceChild(output, prev_output)
```
It should be faster than setting innerHTML to "" and then adding the child

Comment (La0) [reply report.js]: You had 2 `div#output`, one from the html, one created in showDirectory... that was a complicated situation :smirk:

Comment (marco-c) [reply report.js]: Another possibility:
```
    diff --git a/report.js b/report.js
    index 30c3cc1..e325ca7 100644
    --- a/report.js
    +++ b/report.js
    @@ -69,6 +69,7 @@ async function showFile(file) {
       const coverage = await get_path_coverage(file.path);
     
       const table = document.createElement('table');
    +  table.id = 'output';
       table.style.borderCollapse = 'collapse';
       table.style.borderSpacing = 0;
       const tbody = document.createElement('tbody');
```

Comment (La0) [reply report.js]: Did it your way, one line change !

**Review by marco-c — APPROVED** [review]

File: report.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ async function showDirectory(dir, files) {
   files = filter_languages(files);
 
   const columns = [['File name', x => getSpanForFile(x, dir)],
-                   ['Children', x => getSpanForValue('UNKNOWN')],
+                   ['Children', x => getSpanForValue(x.children)],
```
Comment (marco-c) [inline report.js:16]: We should probably show the number of children only for directories

**Pull request merged**